plugins/actionlog/joomla/joomla.php

```diff
@@ -1050,10 +1050,13 @@ public function onAfterPurge($group = 'all')
 	 * Method is called after user update the CMS.
 	 *
 	 * @return  void
+	 * @param   string  $oldVersion  The Joomla version before the update
+	 *
+	 * @return  void
```

Comment:
```suggestion
	 * @param   string  $oldVersion  The Joomla version before the update
	 *
	 * @return  void
```
Ah, my mistake with previous suggestion. Should be only one return value doc comment.